src/contracts/atlas/GasAccounting.sol

```diff
@@ -177,9 +194,19 @@ abstract contract GasAccounting is SafetyLocks {
 
         if (!bidFind) {
```

Comment:
Same as above, `_assign()` is only ever called with bidFind = false. We can remove the param and simplify this a bit

Reply:
https://github.com/FastLane-Labs/atlas/pull/271/commits/51dd8f2b58c343bcc7e47300cbf2bd6be95a1fd6